### PR TITLE
SPADA

### PR DESCRIPTION
**Pull request by john-stoffer**
SPADA Staking Pool

### CODE REVIEW

File: registry/ed25519_pk12wsjyt2zl0de3kslc4n42f6sklhrkzt8vcqyslxn9fnjn2mlad3sp55vue.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (john-stoffer) [inline registry/ed25519_pk12wsjyt2zl0de3kslc4n42f6sklhrkzt8vcqyslxn9fnjn2mlad3sp55vue.json]: This is obsolete, not needed

**Comment by admin-cf-1** [comment]
**Please note that the automated checks are failing.** Passing the automated tests is mandatory. To get more details about why it is failing, click on ‘details’ in the check section and scroll all the way down to the bottom of the resulting page.
(Seems you're touching 4 files, should only be 2...)

File: registry/ed25519_pk1lstlk6evees6jjy5glk6efyvd2uns4rmrrdqcz6amdpls30at0mqg9fx7e.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (john-stoffer) [inline registry/ed25519_pk1lstlk6evees6jjy5glk6efyvd2uns4rmrrdqcz6amdpls30at0mqg9fx7e.json]: How do I resolve the validation signature error?

**Comment by admin-cf-1** [comment]
Closing this PR because it appears to not be longer relevant anymore. If you think this decision was wrong, please comment here so it can be reopened.

**Pull request closed without merging**